accepted/system-range/system-range.md

```diff
@@ -11,19 +11,18 @@ language has to be bound to a .NET type, which we call `System.Range`.
 ### Construcing ranges
 
 ```csharp
-// Ranges are constructed with an index and a length
+// Ranges are constructed with an index and a length:
 
 var fiveToTen = new Range(index: 5, length: 6);
```

Comment:
Why are we using a ctor for length but a factory method for start / end? Instinct is to be consistent and have factory methods in both places.